.circleci/config.yml

```diff
@@ -477,6 +506,29 @@ workflows:
       - run_devtools_e2e_tests:
           requires:
             - build_devtools_and_process_artifacts
+      - run_devtools_tests_for_versions:
+          requires:
+            - build_devtools_and_process_artifacts
+          matrix:
+            parameters:
+              version:
+                - "16.0"
+                - "16.1"
+                - "16.2"
+                - "16.3"
+                - "16.4"
+                - "16.5"
+                - "16.6"
+                - "16.7"
+                - "16.8"
+                - "16.9"
+                - "16.10"
+                - "16.11"
+                - "16.12"
+                - "16.13"
+                - "16.14"
+                - "17.0"
+                - "18.0"
```

Comment:
Seems excessive to run these on every minor. That's not going to scale well in the future, especially if it runs on every PR. Can we maintain a subset of minors that are known to include significant changes that might affect DevTools? Instead of defaulting to every single one.

Similarly I wonder if we really need to run these on every single PR. Maybe we can run them in a cron job workflow instead, like once per hour. With a manual opt-in for specific PRs (like by matching on the commit message, or a GitHub label perhaps).

Reply:
Yeah definitely! I was planning on putting them in a cron workflow and only running these on a subset of minors. 

This draft is just to test the circle CI config and the tests to make sure it works (like it does on my local machine) since I can't setup running circle CI locally. I'm planning on refactoring a lot of this and breaking it up into separate PRs once I get it running correctly!

Reply:
Awesome!